conf/configuration.go

```diff
@@ -34,6 +34,7 @@ type configOptions struct {
 	ImageCacheSize          string
 	AutoImportPlaylists     bool
 	PlaylistsPath           string
+	AutoTranscodeDownlaod   bool
```

Comment:
"Download" is spelled incorrectly here
```suggestion
	AutoTranscodeDownload   bool
```